internal/data/opentelemetry-proto-gen/metrics/v1/metrics.pb.go

```diff
@@ -62,7 +62,7 @@ const (
 	//      number of requests received over the interval of time t_0+1 to
 	//      t_0+2 with a value of 2.
 	AggregationTemporality_AGGREGATION_TEMPORALITY_DELTA AggregationTemporality = 1
-	// CUMULATIVE is an AggregationTemporality for a metic aggregator which
+	// CUMULATIVE is an AggregationTemporality for a metric aggregator which
```

Comment:
How did this file end up being modified? The content of this file is generated from internal/data/opentelemetry-proto and should not change unless the submodule in internal/data/opentelemetry-proto is updated.

I am updating opentelemetry-proto to 0.6.0 right now in https://github.com/open-telemetry/opentelemetry-collector/pull/2031.

To use the new summary data type you will need to first merge the PR that adds the summary data type, then make a new release of opentelemetry-proto and then use the newly released version here.

Reply:
I originally copied the proto changes into the submodule and manually generated the proto to work on the rest of the implementation (while waiting on the proto changes to be merged). 

I will make a new release of the opentelemetry-proto and update this PR when the proto changes merged.

Reply:
Bogdan mentioned in the [proto PR](https://github.com/open-telemetry/opentelemetry-proto/commit/8ab21e9da6246e465cd9d50d405561aedef31a1e) that it isn't necessary to make a new release, but I could just point the submodule to the last commit. I believe this PR is updated with the proto changes now.